tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -579,15 +579,15 @@ EOT
 
 		agent_pkg="${RUST_AGENT_PKG}"
 		agent_dir="${script_dir}/../../../src/agent/"
-		# For now, rust-agent doesn't support seccomp yet.
-		SECCOMP="no"
+		# The kata agent enables seccomp by default.
+		SECCOMP="yes"
```

Comment:
This could look scary to some people :) I would explain in a comment that although seccomp is "enabled", it's not _enforced_ by default: you need to enable that in the main configuration file.

Reply:
I fixed it. Thanks.